pyvista/plotting/plotting.py

```diff
@@ -1950,11 +1950,13 @@ def update(self, stime=1, force_redraw=True):
                     # Execution resumed after TerminateApp() call in on_timer callback, destroy the timer
                     self.iren.destroy_timer(Plotter.update_timer_id)
                 # Rerender
+                self.renderers.update()
                 self.render()
```

Comment:
If using `render()`, then this added line should be unnecessary as the plotter's `render()` call should propagate